### DISCUSSION TITLE
HTMLHelper or Asset Manager

### DISCUSSION THREAD

**Discussion by ghost** [joomla/joomla-cms#32242] [Q&A] [answered]
What is the recommended way to load assets? Especially jquery?

HTMLHelper('bootstrap.framework');

or 

set a dependency in joomla.asset.json of own extensions?

As Joomla not uses jQuery any more. Do we need the row any more:

https://github.com/joomla/joomla-cms/blob/efac4cda1af579c18c8a1e94828aff9da05d9b2d/build/build-modules-js/settings.json#L375

**Comment by Fedik** [comment, marked as answer]
In general, for load an assets use Asset Manager https://docs.joomla.org/J4.x:Web_Assets for Joomla 4

HTMLHelper('bootstrap.framework') it is a helper, that internally uses Asset Manager. You still can use it also.

**Reply by ghost** [reply]
> In general, for load an assets use Asset Manager https://docs.joomla.org/J4.x:Web_Assets for Joomla 4
> 
> HTMLHelper('bootstrap.framework') it is a helper, that internally uses Asset Manager. You still can use it also.

Thank you.

**Comment by dgrammatiko** [comment]
Especially jQuery is something you shouldn’t be using in 2021. Just saying...

**Reply by ghost** [reply]
> Especially jQuery is something you shouldn’t be using in 2021. Just saying...

:)
